notebooks/test_set_creation.ipynb

```diff
@@ -0,0 +1,7095 @@
+{
```

Comment:
<p>Best not make the token publicly viewable, I've used https://pypi.org/project/python-dotenv/ for getting around this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>